### PR TITLE
161780578 Article Reporting

### PR DESCRIPTION
**Pull request by kossy360**
#### Description
Add article reporting feature that enables users to report an article.

#### Type of change
- Feature
  - add article reporting
  - add article reporting validation

#### Pivotal Tracker
#161780578

#### Flow chart
article reporting

#### Postman Tests
- Reporting an article
Screenshot 2019-04-12 at 6 40 47 PM
- Getting all reported articles
Screenshot 2019-04-12 at 6 45 16 PM
- Removing an article from the report table
Screenshot 2019-04-12 at 6 46 05 PM

### CODE REVIEW

File: src/routes/api/articles/reportArticles.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable require-jsdoc */
```
Comment (kayroy247) [inline src/routes/api/articles/reportArticles.js]: Kindly remove this comment. You may need to set that rule in the eslint configuration file.

**Review by kayroy247 — CHANGES_REQUESTED** [review]
Kindly remove the comment in this code.

File: src/routes/api/articles/reportArticles.js (hunk 1 above)
Comment (kossy360) [reply src/routes/api/articles/reportArticles.js]: js-doc descriptions have been added to the file

**Review by kayroy247 — APPROVED** [review]

File: src/models/reportedArticles.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const reportArticles = sequelize.define('reportedArticles', {
+    reason: {
+      type: DataTypes.STRING,
```
Comment (profmcdan) [inline src/models/reportedArticles.js:11]: Add an id field to the model and set it as PK with type UUIDv4

**Review by profmcdan — CHANGES_REQUESTED** [review]
Implement the given feedback

File: src/models/reportedArticles.js (hunk 2 above)
Comment (kossy360) [reply src/models/reportedArticles.js:11]: an id field has been added to the reported articles table

**Review by profmcdan — APPROVED** [review]

**Comment by fuchodeveloper** [comment]
Looks good!

**Pull request merged**